client/examples/million_accounts_genesis.rs

```diff
@@ -40,23 +40,19 @@ fn main_genesis() {
 
     let chain_id = get_chain_id();
     let configuration = get_config(
-        unique_vec![peer.id.clone()],
+        &unique_vec![peer.id.clone()],
         Some(chain_id.clone()),
         Some(get_key_pair()),
     );
     let rt = Runtime::test();
-    let genesis = GenesisNetwork::new(generate_genesis(1_000_000_u32), &chain_id, &{
-        let private_key = configuration
+    let genesis = GenesisNetwork::new(
+        generate_genesis(1_000_000_u32),
+        &chain_id,
+        configuration
             .genesis
-            .private_key
-            .as_ref()
-            .expect("Should be from get_config");
-        KeyPair::new(
-            configuration.genesis.public_key.clone(),
-            private_key.clone(),
-        )
-        .expect("Should be a valid key pair")
-    })
+            .key_pair()
+            .expect("should be available in the config; probably a bug"),
```

Comment:
Because genesis config might be either `Partial` or `Full`. `key_pair()` returns an `Option<KeyPair>`.

In this particular case it should never fail because the config was constructed with a genesis key pair, therefore it's always `Full` there.

Reply:
After #4225 we will no longer circulate private key to special nodes so I assume this will be taken care of

Reply:
I really with #4225 to be merged soon, as it will make the config less ambiguous and easier to document for users.